shared/templates/static/bash/file_permissions_sshd_private_key.sh

```diff
@@ -1,3 +1,8 @@
 # platform = multi_platform_rhel
 
-chmod 0640 /etc/ssh/*_key
+#check if files exist before attempting 
+#to change them
+if ls /etc/ssh/*_key &> /dev/null
+then
+    chmod 0640 /etc/ssh/*_key
+fi
```

Comment:
ah yes... you are quite right, and come to think of it, I tried this a few years ago. /me needs more coffee.

IMO. It is valid for this to fail if the key does not exist even on new install. Passing it would give a false sense of security as well as make a user believe that something was fixed when in fact it was not.